src/main/java/com/crystal/Monster.java

```diff
@@ -26,4 +27,55 @@ public void rankMonster() {
             System.out.println("all-seeing behemoth");
         }
     }
+
+
+    @Override
+    public String getName() {
+        return this.name;
+    }
+
+    @Override
+    public String getClassName() {
+        return "Monster";
+    }
+
+    @Override
+    public boolean isScared() {
+        return scared;
+    }
+
+    @Override
+    public void setScared(boolean scared) {
+        this.scared = scared;
+    }
+
+    @Override
+    public void isInjured(String attack) {
+
+    }
```

Comment:
this should return a boolean given the name?

Reply:
renamed to be unambiguous